tests/vite-app/src/main.ts

```diff
@@ -0,0 +1,49 @@
+import "./style.css";
+import { JwtRsaVerifier } from "aws-jwt-verify";
+
+const inputjwt = document.querySelector<HTMLInputElement>("#jwt");
+const inputissuer = document.querySelector<HTMLInputElement>("#issuer");
+const inputaudience = document.querySelector<HTMLInputElement>("#audience");
+const inputjwskuri = document.querySelector<HTMLInputElement>("#jwksuri");
+const button = document.querySelector<HTMLButtonElement>("#verifyrsa");
+const result = document.querySelector<HTMLSpanElement>("#result");
+const prettyprint = document.querySelector<HTMLPreElement>("#prettyprint");
+
+if (inputjwt) {
+  inputjwt.onkeyup = () => {
+    if (inputjwt && button) {
+      button.disabled = inputjwt.value === "";
+    }
+    if (result) {
+      result.innerHTML = "Unverified";
+    }
+  };
+}
+
+if (button) {
+  button.onclick = async () => {
+    if (inputjwt && inputissuer && inputjwskuri && inputaudience && result) {
+      const verifier = JwtRsaVerifier.create({
+        issuer: inputissuer?.value,
+        audience: inputaudience?.value === "" ? null : inputaudience.value,
```

Comment:
Note to self: check if we have a test with an audience

Reply:
added audience check